### PR TITLE
docs: fix doc about `pass_host` of upstream

### PR DESCRIPTION
**Pull request by nic-chen**
### What this PR does / why we need it:

 fix doc about `pass_host` of upstream

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/zh/latest/admin-api.md [hunk 1]
```diff
@@ -528,7 +528,7 @@ Date: Thu, 26 Dec 2019 08:17:49 GMT
...
-| -------------- | ---------------------------------- | -------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------ | --- |
+| -------------- | ---------------------------------- | -------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------ |
```
Comment (juzhiyuan) [inline docs/zh/latest/admin-api.md:531]: Just to be sure, no need for changes here?

**Comment by Firstsawyou** [comment]
The update content of these two PRs is duplicated. https://github.com/apache/apisix/pull/3805

**Comment by nic-chen** [comment]
> The update content of these two PRs is duplicated. https://github.com/apache/apisix/pull/3805

sorry, I don't see that PR.

You can refer to this PR to see if we could optimize it more.

**Comment by Firstsawyou** [comment]
> > The update content of these two PRs is duplicated. #3805
> 
> @Firstsawyou
> sorry, I don't see that PR.
> 
> You can refer to this PR to see if we could optimize it more.

Okay, thank you for this PR.

**Pull request closed without merging**